src/pq_flash_index.cpp

```diff
@@ -895,16 +908,17 @@ namespace diskann {
                                                             const _u64 n_ids,
                                                             float *dists_out) {
       diskann::aggregate_coords(ids, n_ids, this->data, this->n_chunks,
-                         pq_coord_scratch);
+                                pq_coord_scratch);
       diskann::pq_dist_lookup(pq_coord_scratch, n_ids, this->n_chunks, pq_dists,
-                       dists_out);
+                              dists_out);
     };
-    Timer                 query_timer, io_timer, cpu_timer;
-    std::vector<Neighbor> retset(l_search + 1);
-    tsl::robin_set<_u64> &visited = *(query_scratch->visited);
+    Timer query_timer, io_timer, cpu_timer;
+
+    tsl::robin_set<_u64> & visited = query_scratch->visited;
```

Comment:
nit: extra space before 'v'?

Reply:
that's how clang format is resolving it, so let's leave it as is?

Reply:
ah, then let's leave as is.